pkgs/tools/misc/poop/default.nix

```diff
@@ -2,7 +2,7 @@
 , stdenv
 , fetchFromGitHub
 , fetchpatch
-, zigHook
+, zig
```

Comment:
My experience so far is teaching me that there is not going to be a forwards-compatible Zig introduced any time soon

Maybe these should all be explicit versions so that we don't have to change N packages the next time Zig has a release and we want to change the default Zig version. Maybe to put it another way ... Maybe there's no value yet in having a "default version" of Zig

Reply:
Let's pick a common ground:
The Nix packages should set Zig version explicitly, **but** for the end user it is cool to fire a `nix shell .#pkgs zig`.

Reply:
good idea, I pushed the change as a separate commit